ipfs-cluster-ctl/main.go

```diff
@@ -81,7 +82,12 @@ func main() {
 		cli.StringFlag{
 			Name:  "host, l",
 			Value: defaultHost,
-			Usage: "multiaddress of the IPFS Cluster service API",
+			Usage: "multiaddress of the IPFS Cluster REST API HTTP or LibP2P endpoint",
```

Comment:
Right now this is a little misleading because both libp2p + default endpoints serve HTTP.

Reply:
Any ideas? I'm not sure how to explain this in a 120 char line.. (it's better explained above).